tests/test_repo/oca_dependencies.txt

```diff
@@ -0,0 +1,5 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+
+partner-contact https://github.com/OCA/partner-contact 8.0
```

Comment:
This should be also `partner-contact` to show a simple example. Then, you can put another line with a complete example for a personal repository.